bin/yaml/ruby.yaml

```diff
@@ -0,0 +1,27 @@
+compilers:
+  ruby:
+    yarv:
+      type: tarballs
+      compression: xz
+      dir: ruby-{name}
+      check_exe: bin/ruby --version
+      after_stage_script:
+        # taken from ghc
+        - mkdir install
+        - cd {dir}
+        - ./configure --prefix={destination}/{dir}
+        - make
+        - make install DESTDIR=`cd ../install && pwd`
+        - cd ..
+        - rm -rf {dir}
+        - mv install/{destination}/{dir} {dir}
+        - rm -rf install
+      targets:
+        - name: 2.5.9
+          url: https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.9.tar.xz
```

Comment:
I wish I'd done proper templating here, and then we could do some string manipulation here instead... thanks for working out this is how it has to be done right now.